lib/resolve-bindings-web.js

```diff
@@ -0,0 +1,9 @@
+function resolveBindings(){
+  if (navigator.bluetooth && !process.env.NOBLE_WEBSOCKET) {
+    return require('./webbluetooth/bindings');
+  }
+
+  return require('./websocket/bindings');
```

Comment:
Just a follow up on Gitter, at least for browserify, the web socket bindings will always get included unless there is a rule to exclude them. I think that's why you get a larger bundle size.